### ISSUE TITLE
Error while installing the hound

### ISSUE THREAD

**Issue by Trivikramreddy** [it-projects-llc/hound#46]
```
GOPATH=`pwd` go get github.com/itpp-labs/hound/cmds/...
# github.com/itpp-labs/hound/ui
src/github.com/itpp-labs/hound/ui/ui.go:111:15: undefined: strings.ReplaceAll
src/github.com/itpp-labs/hound/ui/ui.go:251:15: undefined: strings.ReplaceAll


```

**Comment by yelizariev** [comment]
You need to update  go to 1.12+

**Comment by yelizariev** [comment]
I decided to maintain earlier versions of go. The update will be merged with 3.0.0 release
